ci/debian/centreon-ha-common.install

```diff
@@ -0,0 +1,5 @@
+etc/centreon-ha/*                   etc/centreon-ha
+etc/sudoers.d/centreon-cluster-db   etc/sudoers.d
+ocf-scripts/mysql-centreon          usr/lib/ocf/resource.d/heartbeat
```

Comment:
Sorry @lgcosta I don't speak about this.
Now it's now mysql-centreon anymore but there is two files
I just finish my PR yesterday on centreon-buidl and that's why you didn't saw it:
```
# ocf-scripts
%{__install} -d %buildroot%{_prefix}/lib/ocf/resource.d/heartbeat
%{__install} -d %buildroot%{_prefix}/lib/ocf/lib/heartbeat
%{__cp} -r ocf-scripts/mariadb-centreon %buildroot%{_prefix}/lib/ocf/resource.d/heartbeat/
%{__cp} -r ocf-scripts/mariadb-centreon-common.sh %buildroot%{_prefix}/lib/ocf/lib/heartbeat/
```

Reply:
fixed ! Thanks for review !